packages/core-ethereum/src/channel.ts

```diff
@@ -61,13 +61,11 @@ class Channel {
 
     const ticket = unacknowledgedTicket.ticket
 
-    if (ticket.isWinningTicket(await this.commitment.getCurrentCommitment(), response, ticket.winProb)) {
-      const ack = new AcknowledgedTicket(
-        ticket,
-        response,
-        await this.commitment.getCurrentCommitment(),
-        unacknowledgedTicket.signer
-      )
+    // @TODO use some caching to optimize execution time
```

Comment:
this optimisation would live in commitment anyway, so this comment is not relevant.

Reply:
please remove the comment.